pajbot/managers/command.py

```diff
@@ -464,4 +473,13 @@ def parse_command_arguments(
         if "tokens_cost" in options:
             options["tokens_cost"] = abs(options["tokens_cost"])
 
+        if "command_state" in options:
+            state = options["command_state"].lower()
+            if state == "true":
+                options["command_state"] = True
+            elif state == "false":
+                options["command_state"] = False
+            else:
+                options["command_state"] = None
```

Comment:
Where `command_state` is not set it should be assumed to be `True`, since before there was no way to disable a command from chat.